k9mail-library/src/main/java/com/fsck/k9/mail/internet/MimeMessageHelper.java

```diff
@@ -0,0 +1,52 @@
+package com.fsck.k9.mail.internet;
+
+
+import com.fsck.k9.mail.Body;
+import com.fsck.k9.mail.Message;
+import com.fsck.k9.mail.MessagingException;
+import com.fsck.k9.mail.Multipart;
+import com.fsck.k9.mail.Part;
+import org.apache.james.mime4j.util.MimeUtil;
+
+
+public class MimeMessageHelper {
+    private MimeMessageHelper() {
+    }
+
```

Comment:
There's no unit test specifically for this class yet. But because the code was extracted from `Part` it is indirectly tested by some other tests, e.g. `MessageTest`.